scripts/build.ps1

```diff
@@ -107,15 +118,22 @@ $SrcDir = Join-Path $RootDir "src"
 $ArtifactsDir = $null
 $BuildDir = $null
 if ($IsWindows) {
-    $ArtifactsDir = Join-Path $BaseArtifactsDir "windows"
-    $BuildDir = Join-Path $BaseBuildDir "windows"
+    if ($IsUwp) {
+        $ArtifactsDir = Join-Path $BaseArtifactsDir "windows-uwp"
+        $BuildDir = Join-Path $BaseBuildDir "windows-uwp"
```

Comment:
I might prefer just `uwp` instead. It already stands for "Universal Windows Platform" (I think) so an extra windows seems redundant.